Khan/perseus
# Move RadioChoiceWithMetadata from types.js to widget.jsx

## Summary:
---> Not merging until Perseus release complete <----

Based on feedback provided in a [previous PR](https://github.com/Khan/perseus/pull/313), this PR moves the RadioChoiceWithMetadata type from the general types.js file to the widget.jsx file since it is only used by Radio. It also updates the import statement in radio.jsx to reflect the new location.

Issue: LC-161

## Test plan:
- Confirm all tests pass
- Confirm all checks pass
- Confirm no user-facing changes
